build.sbt

```diff
@@ -25,7 +25,9 @@ libraryDependencies ++= Seq(
   "com.microsoft.cognitiveservices.speech" % "client-sdk" % "1.8.0",
   "org.apache.httpcomponents" % "httpclient" % "4.5.6",
   "com.microsoft.ml.lightgbm" % "lightgbmlib" % "2.3.150",
-  "com.github.vowpalwabbit" %  "vw-jni" % "8.7.0.3"
+  "com.github.vowpalwabbit" % "vw-jni" % "8.7.0.3",
+  "com.linkedin.isolation-forest" %% "isolation-forest_2.4.3" % "0.3.1",
```

Comment:
0.3.2 is the version in Maven Central.

https://search.maven.org/search?q=isolation-forest